app/assets/javascripts/modules/soloteam.js.coffee

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2008-2016, Puzzle ITC GmbH. This file is part of
+# Cryptopus and licensed under the Affero General Public License version 3 or later.
+# See the COPYING file at the top-level directory or at
+# https://github.com/puzzle/cryptopus.
+
+app = window.App ||= {}
+
+class app.Soloteam
+  @hideSoloteams: (teams_div) ->
```

Comment:
Warum verzieren wir die Methodendefinitionen in dieser Klasse mit '@'?
In account_move.js.coffee machen wir das nicht.